lib/index.d.ts

```diff
@@ -2254,3 +2287,119 @@ declare module 'react-native-firebase' {
     }
   }
 }
+
+declare module 'react-native-firebase/storage' {
+  import { RNFirebase } from 'react-native-firebase';
+  export type StorageTask<T> = RNFirebase.storage.StorageTask<T>;
+  export type RNStorage = RNFirebase.storage.RNStorage;
+  export type Storage = RNFirebase.storage.Storage;
+  export type Reference = RNFirebase.storage.Reference;
+  export type UploadMetadata = RNFirebase.storage.UploadMetadata;
+  export type SettableMetadata = RNFirebase.storage.SettableMetadata;
+  export type StringFormat = RNFirebase.storage.StringFormat;
+  export type UploadTask = RNFirebase.storage.UploadTask;
+  export type UploadTaskSnapshot = RNFirebase.storage.UploadTaskSnapshot;
+  export type FullMetadata = RNFirebase.storage.FullMetadata;
+  export type TaskEvent = RNFirebase.storage.TaskEvent;
+  export type TaskState = RNFirebase.storage.TaskState;
+}
+
+declare module 'react-native-firebase/database' {
+  import { RNFirebase } from 'react-native-firebase';
+  export type Database = RNFirebase.database.Database;
+  export type RnReference = RNFirebase.database.RnReference;
```

Comment:
Not sure why `RnReference` needs separate definitions from `Reference`, should whats in `RnReference` not be a part of `Reference`? 

Appreciate that it was already defined there before this PR so probably out of scope of this PR. Just wanted to get your thoughts